scripts/sdk/Fuse.ts

```diff
@@ -0,0 +1,261 @@
+  };
+
+  public getPoolsByAccount = async (address: string) => {
+    const [poolIndexes, poolDescriptions] =
+      await this.contracts.FusePoolDirectory.functions.getPoolsByAccount(
+        address
+      );
+
+    return {
+      poolIndexes,
+      poolDescriptions,
+    };
+  };
+
+  public getPublicPoolsByVerification = async () => {
+    const [poolIndexes, poolDescriptions] =
+      await this.contracts.FusePoolDirectory.functions.getPublicPoolsByVerification(
+        true
+      );
+
+    return {
+      poolIndexes,
+      poolDescriptions,
+    };
+  };
+
+  // Admin
+
+  public pauseAllBorrowableTokensByIndex = async (index: number) => {
+    const { name, comptroller, borrowableAssets } =
+      await this.getBorrowableAssetsByIndex(index);
+
+    console.warn(`Pausing all borrowable assets in pool ${index}: ${name}`);
+    console.log(`Comptroller address: ${comptroller}`);
+
+    console.log(borrowableAssets);
+  };
+
+  // Happens when someone is able to withdraw past 100% utilization rate because of accumulated Fuse fees/reserves
+  public getPoolsWithTooHighBorrowRate = async () => {
+    // Maximum borrow rate that can ever be applied (.0005% / block)
+    const borrowRateMaxMantissa = BigNumber.from(0.0005e16);
+
+    const { poolDescriptions } = await this.getPublicPoolsByVerification();
+
+    const checkBorrowRate = async (pool: any) => {
+      const [name, , comptroller] = pool;
+
+      const cTokens = (
+        await this.getAllMarketsByComptroller(comptroller)
+      ).flat();
+
+      return await Promise.all(
+        cTokens.map(async (market: string) => {
+          const borrowRate = BigNumber.from(
+            (
+              await this.getCToken(market).functions.borrowRatePerBlock()
+            ).toString()
+          );
+
+          if (!borrowRate.gte(borrowRateMaxMantissa)) {
+            return false;
+          }
+
+          return {
+            name,
+            comptroller,
+            market,
+            borrowRate: borrowRate.toString(),
+          };
+        })
+      );
+    };
+
+    const results = (
+      await promiseAllBatch(checkBorrowRate, poolDescriptions, 3)
+    )
+      .flat()
+      .filter(Boolean);
+
+    return results;
+  };
+
+  public getCalldataBorrowRateTooHigh = async (ctokenAddress: string) => {
+    const CTOKEN_IRM_FIX_IMPLEMENTATION =
+      "0x46f196f21f420e3ea159b706d249046e80f05f7e";
+
+    const CTOKEN_DEFAULT_IMPLEMENTATION =
+      "0x67db14e73c2dce786b5bbbfa4d010deab4bbfcf9";
+
+    const cToken = this.getCErc20Delegate(ctokenAddress);
+
+    const targets = [ctokenAddress, ctokenAddress];
+
+    const data = [
+      cToken.interface.encodeFunctionData("_setImplementationSafe", [
+        CTOKEN_IRM_FIX_IMPLEMENTATION,
+        false,
+        0x0,
+      ]),
+      cToken.interface.encodeFunctionData("_setImplementationSafe", [
+        CTOKEN_DEFAULT_IMPLEMENTATION,
+        false,
+        0x0,
+      ]),
+    ];
+
+    return [targets, data];
```

Comment:
@sriyantra this would log the calldata as follows

```
[
  [
    '0x647A36d421183a0a9Fa62717a64B664a24E469C7',
    '0x647A36d421183a0a9Fa62717a64B664a24E469C7'
  ],
  [
    '0x50d85b7300000000000000000000000046f196f21f420e3ea159b706d249046e80f05f7e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000',
    '0x50d85b7300000000000000000000000067db14e73c2dce786b5bbbfa4d010deab4bbfcf90000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000'
  ]
]
```

I don't think this is exactly what you are looking for though